src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -25,6 +25,7 @@
     </cd-table>
   </tab>
   <tab i18n-heading
+       *ngIf="grafanaScope.read"
```

Comment:
Since this component is already has a local var for permissons,  you could:
```suggestion
       *ngIf="permissions.grafana.read"
```

Reply:
ack